conf/app.routes

```diff
@@ -32,6 +32,12 @@ POST       /:draftId/select-valuation-method                     controllers.Val
 GET        /:draftId/change-select-valuation-method               controllers.ValuationMethodController.onPageLoad(mode: Mode = CheckMode, draftId: DraftId)
 POST       /:draftId/change-select-valuation-method               controllers.ValuationMethodController.onSubmit(mode: Mode = CheckMode, draftId: DraftId)
 
+GET        /:draftId/provide-trader-eori                                          controllers.ProvideTraderEoriController.onPageLoad(mode: Mode = NormalMode, draftId: DraftId)
+POST       /:draftId/provide-trader-eori                                          controllers.ProvideTraderEoriController.onSubmit(mode: Mode = NormalMode, draftId: DraftId)
+
+GET        /:draftId/verify-trader-eori                                          controllers.VerifyTraderEoriController.onPageLoad(mode: Mode = NormalMode, draftId: DraftId)
```

Comment:
Isn't this page is is **/:draftId/check-EORI-details**

Reply:
We can resolve the duplication at standup

Reply:
Agreement was to ignore this for now